azavea/pfb-network-connectivity
# Update methodology page content

## Overview

Update content of methodology page.


### Demo

![image](https://user-images.githubusercontent.com/960264/51706927-df905880-1fed-11e9-8bdc-982de767d23d.png)


### Notes

The source document uses two different symbols interchangeably for apostrophes. All should be replaced in the HTML with `&#39;`.


## Testing Instructions

 * Check the [methodology page](http://localhost:9301/#/methodology)
 * Content should match source document
 * Links should work and open in a new tab if pointing to a site
 * Apostrophes should be represented as expected


Closes #667.
